src/ORM/Locator/LocatorInterface.php

```diff
@@ -18,12 +18,31 @@
 
 /**
  * Registries for Table objects should implement this interface.
- *
- * @method array getConfig($alias)
- * @method $this setConfig($alias, $options = null)
  */
 interface LocatorInterface
 {
+    /**
+     * Returns configuration for an alias or the full configuration array for
+     * all aliases.
+     *
+     * @param string|null $alias Alias to get config for, null for complete config.
+     * @return array The config data.
+     */
+    public function getConfig($alias = null);
+
+    /**
+     * Stores a list of options to be used when instantiating an object
+     * with a matching alias.
+     *
+     * @param string|array $alias Name of the alias or array to completely
+     *   overwrite current config.
+     * @param array|null $options list of options for the alias
+     * @return $this
+     * @throws \RuntimeException When you attempt to configure an existing
+     *   table instance.
+     */
+    public function setConfig($alias, $options = null);
```

Comment:
can options be an array now?

Reply:
Probably, but I am gonna leave that for later when we start adding proper typehints across the framework.